### ISSUE TITLE
0.8.1 release for fixing nightly Rust

### ISSUE THREAD

**Issue by cquintana-verbio** [zslayton/cron#85]
Hi!

First of all, thanks for your effort in this crate. We've been using it for a while with great success!

However, since the latest nightly we've been experiencing some problems with it:

```
error[E0432]: unresolved import `std::collections::Bound`
 --> /opt/app-root/src/.cargo/registry/src/github.com-1ecc6299db9ec823/cron-0.8.0/src/schedule.rs:6:23
  |
6 | use std::collections::Bound::{Included, Unbounded};
  |                       ^^^^^ `Bound` is a type alias, not a module
error: aborting due to previous error
```

It looks like the deprecated alias has been removed. 

We saw that the problem has been fixed by https://github.com/zslayton/cron/pull/84 . Would it be possible to make a `0.8.1` release that contains this change?

Thank you!

**Comment by zslayton** [comment]
[Version `0.9.0`](https://github.com/zslayton/cron/releases/tag/v0.9.0) is now available; it includes the fix from #84.

I'm glad you've found it useful! Out of interest, what project/crate are you using it in?

**Comment by cquintana-verbio** [comment]
We're using it in an internal company development, and we have a task scheduler that accepts the `cron` format as the schedule definition :)

Thanks a lot!
